todo_app/e2etests/test_e2e.py

```diff
@@ -0,0 +1,76 @@
+import os
+import pytest
+
+from threading import Thread
+from todo_app import app
+from todo_app.services import trello
+from dotenv import find_dotenv, load_dotenv
+from selenium import webdriver
+from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+@pytest.fixture(scope="module")
+def driver():
+    os.environ["PATH"] += os.pathsep + '/Users/paulhutchinson/Dev/Corndel/Course/ex2/DevOps-Course-Starter/todo_app/e2etests'
```

Comment:
If you are keeping it in this folder you probably want to include geckodriver in your gitignore so you don't accidentally commit it  (or don't keep it in the repository)